### PR TITLE
Answer 22: Angular router input

### PR DESCRIPTION
**Pull request by Clarchik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
     "postcss-import": "~14.1.0",
     "postcss-preset-env": "~7.5.0",
     "postcss-url": "~10.1.3",
-    "prettier": "^2.6.2",
+    "prettier": "^3.1.0",
```
Comment (geromegrignon) [inline package.json:95]: While important to update the dependencies of a project, a commit should focus on its goal. Such a change should land in a dedicated commit.

Comment (tomalaforge) [reply package.json:95]: you can create a PR if you want to upgrade prettier, but a global update will happen soon.
